src/containers/activity/activity-area.tsx

```diff
@@ -0,0 +1,126 @@
+import Anchor from '@ui/anchor';
+import { getTimeAgo } from '@utils/getTimeAgo';
+import clsx from 'clsx';
+import Image from 'next/image';
+import useSWR from 'swr';
+import { itemService } from '../../features/leda-nft/services/item.service';
+
+type Props = {
+  space?: number;
+  className?: string;
+};
+
+const fetchHistories = () => itemService.findAllHistory().then((data) => data.reverse());
+
+export const ActivityArea = ({ space = 1, className }: Props) => {
+  const { data: history } = useSWR('fetchAllHistory', fetchHistories);
+  // TODO: Enable filters if needed
+  // const marketFilters = [
+  //   ...new Set(flatDeep(data?.activities.map((activity) => activity.marketFilters))),
+  // ];
+  // const userFilters = [
+  //   ...new Set(flatDeep(data?.activities.map((activity) => activity.userFilters))),
+  // ];
+
+  // const filterHandler = (filter) => {
+  //   const newActivities = data?.activities.filter(
+  //     (activity) => activity.marketFilters.includes(filter) || activity.userFilters.includes(filter)
+  //   );
+  //   setActivities(newActivities);
+  // };
+
+  if (!history?.length) {
+    return (
+      <div className="text-center my-5">
+        <h3>No history found</h3>
+      </div>
+    );
+  }
+
+  if (!history) {
+    return <>Loading</>;
+  }
+  return (
+    <div className={clsx('rn-activity-area', space === 1 && 'rn-section-gapTop', className)}>
```

Comment:
As far as I know we don't longer need the space and className, so you could drop the first div. Double check first

Reply:
I didn't realize, it was already there